api/graphql/tests/test_reservations/test_reservation_queries.py

```diff
@@ -296,6 +340,149 @@ def test_reservation_total_count(self):
         assert_that(content.get("errors")).is_none()
         self.assertMatchSnapshot(content)
 
+    def test_hide_fields_with_personal_information(self):
+        self.create_reservation_by_admin()
+        self.client.force_login(self.regular_joe)
+        response = self.query(
```

Comment:
Lots of duplicate query string, perhaps add a class method for getting it?
Also could add begin and end fields at least to confirm that reservation time things are visible.

Reply:
Great ideas! Implemented both 🧹